File: CapereVexillum/app/src/main/java/comwim07101993ictproj3_capturetheflag/github/caperevexillum/models/Team.java
package comwim07101993ictproj3_capturetheflag.github.caperevexillum.models;

/**
 * Created by Michiel on 12/10/2017.
 */

public class Team {

    public static String TEAM_GREEN="green";
    public static String TEAM_ORANGE ="orange";
    public static String NO_TEAM = "no_team";

    private String teamName;
    private int score;


    public Team(String teamName, int score){
        this.teamName = teamName;
        this.score = score;
    }


    public String getTeamName() {
        return teamName;
    }

    public void setTeamName(String teamName) {
        this.teamName = teamName;
    }

    public int getScore() {
        return score;
    }

    public void setScore(int score) {
        this.score = score;
    }

}

File: socketapp/db/team.js
export default class Team {
    teamname = "";
    score = "";
    players = [];

    constructor(teamname, score, players){
        this.teamname = teamname;
        this.score = score;
        this.players = players;
    }

    addPlayer(player){
        this.players.push(player);
    }

}